src/DataBus/ConfigureAzureDataBus.cs

```diff
@@ -144,15 +144,15 @@ public static DataBusExtensions<AzureDataBus> CleanupInterval(this DataBusExtens
         }
 
         /// <summary>
-        /// Set the BlobContainerClient to use to control authentication against the service.
+        /// Set the BlobServiceClient to use to control authentication against the service.
```

Comment:
It's not just about the authentication, right? The entire HTTP pipeline behind it as well.
I wonder if this line should be simpler.
```suggestion
        /// Set the custom <see cref="BlobServiceClient"/> to be used by the persistence, enabling the necessary customizations to the client.
```